README.md

```diff
@@ -155,7 +154,6 @@ The genezio CLI tool supports the commands shown in the following table:
 
 | Command                                        | Description                                                                                                                                                                                                  |
 | ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Add `genezio` and `genezio create` commands. Copy the content from this page: https://docs.genez.io/genezio-dev-documentation/~/changes/0mNx5jxxocT3LaouL9eE/cli-tool/cli-commands